metadata-service/war/src/main/resources/boot/policies.json

```diff
@@ -140,5 +140,264 @@
+        "groups":[
+          "urn:li:corpgroup:metadata-stewards"
+        ]
+      },
+      "privileges":[
+        "MANAGE_INGESTION",
+        "MANAGE_SECRETS",
+        "VIEW_ANALYTICS",
+        "GENERATE_PERSONAL_ACCESS_TOKENS",
+        "MANAGE_DOMAINS",
+        "MANAGE_TESTS",
+        "MANAGE_GLOSSARIES",
+      ],
+      "displayName":"Metadata Stewards - Platform Policy",
+      "description":"Metadata Stewards can manage ingestion and view analytics.",
+      "state":"ACTIVE",
+      "type":"PLATFORM",
+      "editable":false
+    }
+  },
+  {
+    "urn": "urn:li:dataHubPolicy:metadata-stewards-metadata-policy",
+    "info": {
+      "actors":{
+        "resourceOwners":false,
+        "allUsers":false,
+        "allGroups":false,
+        "users":[],
+        "groups":[
+          "urn:li:corpgroup:metadata-stewards"
+        ]
+      },
+      "privileges":[
+        "VIEW_ENTITY_PAGE",
+        "EDIT_ENTITY_TAGS",
+        "EDIT_ENTITY_GLOSSARY_TERMS",
+        "EDIT_ENTITY_OWNERS",
+        "EDIT_ENTITY_DOCS",
+        "EDIT_ENTITY_DOC_LINKS",
+        "EDIT_ENTITY_STATUS",
+        "EDIT_DOMAINS_PRIVILEGE",
+        "EDIT_DEPRECATION_PRIVILEGE",
+        "EDIT_ENTITY_ASSERTIONS",
+        "EDIT_ENTITY_OPERATIONS",
+        "EDIT_ENTITY",
+        "EDIT_DATASET_COL_TAGS",
+        "EDIT_DATASET_COL_GLOSSARY_TERMS",
+        "EDIT_DATASET_COL_DESCRIPTION",
+        "VIEW_DATASET_USAGE",
+        "VIEW_DATASET_PROFILE",
+        "EDIT_TAG_COLOR",
+        "EDIT_GROUP_MEMBERS",
+        "EDIT_USER_PROFILE",
+        "EDIT_CONTACT_INFO"
+      ],
+      "displayName":"Metadata Stewards - Metadata Policy",
+      "description":"Metadata Stewards have all metadata privileges.",
+      "state":"ACTIVE",
+      "type":"METADATA",
+      "editable":false
+    }
+  },
+  {
+    "urn": "urn:li:dataHubPolicy:asset-owners-metadata-policy",
+    "info": {
+      "actors":{
+        "resourceOwners":true,
+        "allUsers":true,
+        "allGroups":false,
+        "users":[],
+        "groups":[]
+      },
+      "privileges":[
+        "VIEW_ENTITY_PAGE",
+        "EDIT_ENTITY_TAGS",
+        "EDIT_ENTITY_GLOSSARY_TERMS",
+        "EDIT_ENTITY_OWNERS",
+        "EDIT_ENTITY_DOCS",
+        "EDIT_ENTITY_DOC_LINKS",
+        "EDIT_ENTITY_STATUS",
+        "EDIT_DOMAINS_PRIVILEGE",
+        "EDIT_DEPRECATION_PRIVILEGE",
+        "EDIT_ENTITY_ASSERTIONS",
+        "EDIT_ENTITY_OPERATIONS",
+        "EDIT_ENTITY",
+        "EDIT_DATASET_COL_TAGS",
+        "EDIT_DATASET_COL_GLOSSARY_TERMS",
+        "EDIT_DATASET_COL_DESCRIPTION",
+        "VIEW_DATASET_USAGE",
+        "VIEW_DATASET_PROFILE",
+        "EDIT_TAG_COLOR",
+        "EDIT_GROUP_MEMBERS",
+        "EDIT_USER_PROFILE",
+        "EDIT_CONTACT_INFO"
+      ],
+      "displayName":"Asset Owners - Metadata Policy",
+      "description":"Asset owners have all metadata privileges ONLY for assets they own.",
+      "state":"ACTIVE",
+      "type":"METADATA",
+      "editable":false
```

Comment:
Since this is NOT editable people will not be able to customize this. Seems dangerous.

Reply:
What if people don't want owners to be able to do certain things?